common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -404,6 +404,32 @@ def test_video_with_youtube_blocked(self):
 
         self.assertTrue(self.video.is_video_rendered('html5'))
 
+    def test_video_with_youtube_blocked_delay(self):
+        """
+        Scenario: Video is rendered in HTML5 mode when the YouTube API is blocked
+                  but we find out after the initial request to YouTube times out
+        Given the YouTube server response time is greater than 1.5 seconds
```

Comment:
The thing that's relevant is that the iframe delay is greater than the time_to_response, right? So should I just add:
`Given the Youtube iframe loading delay is longer than the YouTube server response time`?